_posts/2020-08-26-#19339.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: pr
+date: 2020-08-26
+title: "Redelgate AbsurdFee from Mempool to Clients"
+pr: 19339
+authors: [gzhao408]
+components: ["validation", "mempool", "rpc"]
+host: gzhao408
+status: upcoming
+commit: 67bb714
+---
+
+## Notes
+* [AcceptToMemoryPool()](https://github.com/bitcoin/bitcoin/blob/master/src/validation.cpp#L1084) (ATMP) processes a transaction for acceptance to the mempool. 
+In addition to enforcing consensus rules, it also applies the node’s policy. 
+[#19339](https://github.com/bitcoin/bitcoin/pull/19339) deals with acceptance logic that falls into neither category… and thus doesn’t belong in validation!
+
+* For reference, [Policy](https://github.com/bitcoin/bitcoin/tree/master/src/policy) = a node’s extra standards for validation, only applied to unconfirmed transactions. 
```

Comment:
suggest s/Policy =/policy is|represents/

" only applied to unconfirmed transactions" -> is this true? if true TIL, can you link to a reference?

Reply:
Ok yeah so I don't want to say something that isn't completely accurate; you most certainly know better than me.
My thinking was, when validating confirmed transactions (i.e. blocks) we only apply consensus rules, otherwise we'd be too strict. Policy and "standardness" are not consensus-critical so we don't apply them in block validation because we'd be too strict. There's a [comment](https://github.com/bitcoin/bitcoin/tree/master/src/policy/policy.h#L56) to this effect, although it doesn't say exactly what I'm saying ("only applied to unconfirmed").

I think of `GetBlockScriptFlags()` as the flags used for confirmed but this may be incomplete.

Reply:
> you most certainly know better than me.

I wouldn't assume that :smile: especially in this case, I'd need to look.

Reply:
I changed it to "policy represents a node’s extra standards for validation, typically applied in situations where it may be appropriate to apply rules in addition to consensus."
which may sound a little wishy-washy 😅

Reply:
A quick codebase grep and search online didn't turn up any specific definition of Bitcoin Core "policy" for me. The best so far was just to look in the files in the `src/policy/` dir, especially `policy/policy.h`.

Reply:
Yeah ☹️ I was confused about what policy was for a long time.